jobs/dcp1-version/docker-app/scripts/populate_dcp_version.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import os
+import uuid
+from datetime import datetime
+
+from bson import JAVA_LEGACY, CodecOptions
+from pymongo import MongoClient
+
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+DCP1_GS_FILES_LIST = os.environ.get('DCP1_GS_FILES_LIST')
```

Comment:
If there is no default value, there should probably be validation that the value is populated.

Reply:
Good point, thanks!